plugins/process/src/main/java/org/radargun/service/ProcessLifecycle.java

```diff
@@ -402,4 +399,34 @@ private String getServiceChildPIDs() {
       }
       return "";
    }
+   
+   private boolean isPIDAlive(String pid) {
+      if (pid != null) {
+         ProcessBuilder pb = new ProcessBuilder().command("ps", "-o", "pid=", "-p", pid);
```

Comment:
Agreed, that was the point of a discussion I have with Martin some week or so ago. I was instructed to not bother right now and leave it for the Windows effort later.